test/xpu_api/utilities/meta/meta.trans/meta.trans.other/aligned_storage.pass.cpp

```diff
@@ -0,0 +1,107 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+// type_traits
+
+// aligned_storage
+//
+//  Issue 3034 added:
+//  The member typedef type shall be a trivial standard-layout type.
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/type_traits>
+
+#include "support/test_macros.h"
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+bool
+kernel_test()
+{
+    {
+        typedef dpl::aligned_storage<10, 1>::type T1;
+        ASSERT_SAME_TYPE(T1, dpl::aligned_storage_t<10, 1>);
+
+#if TEST_STD_VER == 17
+        static_assert(dpl::is_pod<T1>::value);
+#endif
```

Comment:
I would add a comment here that `dpl::is_pod` is deprecated in C++20.

Reply:
As we may see, all usages of `dpl::is_pod` is under define `#if TEST_STD_VER == 17` so I believe we haven't problems here...